### DISCUSSION TITLE
Clarify conventions around declaring Class `properties` vs `heritableProperties` in the core-im

### DISCUSSION THREAD

**Discussion by mbrush** [ga4gh/va-spec#153] [General]
Can someone define the rationale for when to use one vs the other was chosen in the current core-im model? 

I suspect if has to do with whether the class has subtypes in the core model  (but the whole profiling framework adds a twist here).

Once the rules are clear, we should review their current usage, and fix any that need fixing?

**Comment by ahwagner** [comment]
This is a metaschema processor convention. `heritableProperties` are used by abstract classes, `properties` by concrete classes. You'll note that when processed to JSON Schema, there are no data classes that retain a `heritableProperties` property.

**Comment by mbrush** [comment]
Thanks Alex - I fixed incorrect usages of heritableProperties in PR #154.
